src/com/facebook/buck/swift/SwiftDescriptions.java

```diff
@@ -54,6 +60,16 @@ private SwiftDescriptions() {
       BuildTarget buildTarget) {
     output.srcs = args.srcs.transform(
         input -> filterSwiftSources(sourcePathResolver, args.srcs.get()));
+    output.exportedHeaders = args.exportedHeaders.transform(
```

Comment:
why does swift has exportedHeaders? isn't it something Apple-specific? Does swift on *nix have it?